.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should be added?

Reply:
Yes. I don't see a reson not to share workspace settings